.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Expected behavior**
+Describe what you expected to happen.
+
+**Actual behavior**
+Describe what actually happened that was unexpected.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**System information**
+Please provide the following information:
+ - SDK Version: 
+ - OS type and version:
+ - Application Server type and version (if applicable):
+ - Using spring-boot?
+ - Additional relevant libraries (with version, if applicable):
+
+**Logs**
+[Turn on SDK logs](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-troubleshoot#debug-data-from-the-sdk) and attach/paste them to the issue. If using an application server, also attach any relevant server logs.
```

Comment:
We probably need to advise on removing any sensitive information from those logs before attaching to public repo on GitHub.

Reply:
+1 yes ... also when sometimes user uploads a repro we should advise to remove all the PIDs

Reply:
Ah, good catch. Will do